bulbs/super_features/utils.py

```diff
@@ -17,39 +18,51 @@ def get_data_serializer(name):
 
 
 def get_superfeature_model():
-    from bulbs.super_features.models import BaseSuperFeature
-
-    model_name = getattr(settings, "BULBS_SUPERFEATURE_MODEL", None)
-    if model_name is None:
+    module = getattr(settings, "BULBS_SUPERFEATURE_MODEL_MODULE", None)
+    model = getattr(settings, "BULBS_SUPERFEATURE_MODEL", None)
+    if model is None or module is None:
+        from bulbs.super_features.models import BaseSuperFeature
         return BaseSuperFeature
-    return apps.get_model(model_name)
+    else:
+        # NOTE: import serializer module & then evaluate based on serializer name
+        mod = import_module(module)  # NOQA
+        return eval("{0}.{1}".format('mod', model))
 
 
 def get_superfeature_serializer():
-    from bulbs.super_features.serializers import BaseSuperFeatureSerializer
-
+    module = getattr(settings, "BULBS_SUPERFEATURE_SERIALIZER_MODULE", None)
     serializer = getattr(settings, "BULBS_SUPERFEATURE_SERIALIZER", None)
-    if serializer is None:
+    if serializer is None or module is None:
+        from bulbs.super_features.serializers import BaseSuperFeatureSerializer
         return BaseSuperFeatureSerializer
-    return serializer
+    else:
+        # NOTE: import serializer module & then evaluate based on serializer name
+        mod = import_module(module)  # NOQA
+        return eval("{0}.{1}".format('mod', serializer))
```

Comment:
`eval()` is basically getting us the actual classes that we want. since we have the module and the string name of the class, we evaluate them together in dot form to get the class.